DatePicker/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="author" content="Matúš Ušiak">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Date picker custom element</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+  <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+
+  <script src="https://app.kenticocloud.com/js-api/custom-element.js"></script>
+  <script>
+
+    function changeSize(size){
+      CustomElement.setHeight(size);
+    }
+  
+  </script>
+</head>
+
+<body>
+  <p>Date: <input type="text" id="datepicker" onfocus="changeSize(300)"/></p>
```

Comment:
The date label is not necessary and I would use dif as a wrapper

```suggestion
  <div>
    <input type="text" id="datepicker" onfocus="changeSize(300)"/>
  </div>
```

Reply:
Ideally, I would include [custom-element.css](https://github.com/Kentico/custom-element-samples/blob/master/shared/custom-element.css) as a part of the pull request.

Linked them as an external style sheet to the head of HTML
```
  <link rel="stylesheet" href="custom-element.css">
```

And then wrapped the input into the div instead of p to have the html semantically correct
Set the width to 270 px to have the input same size as a picker and set it class by the [examples](https://kentico.github.io/custom-element-samples/shared/examples.html)

```suggestion
    <div style="width:270px;">
      <input class="text-field__input" type="text" id="datepicker" onfocus="changeSize(300)"/>
    </div>
```

![image](https://user-images.githubusercontent.com/9218736/55569166-14e0b300-56cf-11e9-873b-72315dac4c4f.png)

![image](https://user-images.githubusercontent.com/9218736/55569175-18743a00-56cf-11e9-86fe-12244098c381.png)